homeassistant/components/denonavr/icons.json

```diff
@@ -0,0 +1,7 @@
+{
+  "services": {
+    "get_command": "mdi:console",
+    "set_dynamic_eq": "mdi:tune",
+    "update_audyssey": "mdi:tune-vertical"
```

Comment:
I have no idea what an audyssey is and if this icon is fitting. So if someone has a better alternative, please let me know

Reply:
Seems to be a microphone that's placed in the room to then "tune the AVR".
IMO, the tune icon is fine. An alternative might be `mdi:microphone`. No real preference from my side at least.

Reply:
Audyssey is the callibration method using a microphone to get the volume level of each frequency adjusted to your specific room and speaker placement.

It gives you a optimezed freqency response.

I personally would like one of these icons:
mdi:waveform
mdi:equalizer-outline

But tune-vertical would also work.

Don't have a strong opinion.